src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/service/LimitedPartnershipService.java

```diff
@@ -49,9 +51,11 @@ public String createLimitedPartnership(Transaction transaction,
         LimitedPartnershipSubmissionDao dao = mapper.dtoToDao(limitedPartnershipSubmissionDto);
         dao.setCreatedAt(LocalDateTime.now());
         dao.setUserId(userId);
+
         LimitedPartnershipSubmissionDao insertedSubmission = repository.insert(dao);
 
         final String submissionUri = getSubmissionUri(transaction.getId(), insertedSubmission.getId());
+        updateLimitedPartnershipubmissionWithMetaData(insertedSubmission, submissionUri); 
```

Comment:
`updateLimitedPartnershipSubmissionWithSelfLink` spelling but I would also be more specific, since at the moment (and maybe until the end of time!) this method is only adding one field.